src/import/sb2import.js

```diff
@@ -138,7 +135,12 @@ var parseScratchObject = function (object, runtime, topLevel) {
         }
     }
     target.isStage = topLevel;
-    Promise.all(costumePromises).then(function () {
+    Promise.all(costumePromises).then(function (costumes) {
+        sprite.costumes = costumes.filter(
+            function (c) {
+                return c !== null;
+            }
+        );
```

Comment:
It looks to me like there's no longer any way to get here with a `null`; maybe you don't need the filter?